docs/report/main.tex

```diff
@@ -404,7 +425,7 @@ \subsection{Gates and Operators}
 \vspace{5mm}
 
 \noindent
-The phase shift gate modifies the phase of the quantum state and is equivalent to tracing a horizontal circle  on the Bloch Sphere(Refer to page 3) by $\varphi$ radians. 
+The phase shift gate modifies the phase of the quantum state and is equivalent to tracing a horizontal circle  on the Bloch Sphere (Refer to Figure 1 on on page 3) by $\varphi$ radians\cite{noauthor_quantum_nodate}. 
```

Comment:
If you add labels to the figure captions, you can use the `\ref` tag to auto number your figure references.

Reply:
Good shout bro. I have added the label in now. I will create PR later when I add more in the report. I am going to write some bit for QFT.

Reply:
I'm working on splitting the figure numbering in the TikZ charts you have into fig 1a/Fig 1b. So could do it as part of a PR for that if you want.

Reply:
sure Tiernan, you are welcome to do that. thanks!

Reply:
https://github.com/Tiernan8r/quantum_computing_project/pull/80 should do it